presto-orc/src/test/java/com/facebook/presto/orc/metadata/TestDwrfMetadataReader.java

```diff
@@ -355,17 +355,19 @@ public Object[][] columnStatisticsSupplier()
         ColumnStatistics integerColumnStatistics1 = new IntegerColumnStatistics(7L, null, new IntegerStatistics(25L, 95L, 100L));
         ColumnStatistics integerColumnStatistics2 = new IntegerColumnStatistics(9L, null, new IntegerStatistics(12L, 22L, 32L));
 
-        MapColumnStatisticsBuilder mapStatsIntKeyBuilder = new MapColumnStatisticsBuilder();
+        MapColumnStatisticsBuilder mapStatsIntKeyBuilder = new MapColumnStatisticsBuilder(true);
```

Comment:
Changed this test to follow up on my mistake in the initial implementation of MapColumnStatisticsBuilder